packages/medusa/src/api/routes/admin/price-lists/delete-product-prices.ts

```diff
@@ -0,0 +1,48 @@
+import PriceListService from "../../../../services/price-list"
+
+/**
+ * @oas [delete] /price-lists/{id}/products/{product_id}
+ * operationId: "DeletePriceListsProductPriceListPrices"
```

Comment:
```suggestion
 * operationId: "DeletePriceListsPriceListProductsProduct"
```

Reply:
The operationIds are not really semantic but are derived directly from the path:

- delete -> Delete
- /price-lists -> PriceLists
- /{id} -> PriceList
- /products -> Products
- /{product_id} -> Product

The operation Id simply has to be unique and this approach ensures that we don't have duplicates; furthermore, it makes it straightforward to construct the path from the operation id.